internal/server/jobs/jobs.go

```diff
@@ -0,0 +1,20 @@
+package jobs
```

Comment:
Do we need a separate package for this? This seems like a pretty natural fit for the `internal/server` package, since these are jobs performed by our api server.

Reply:
Perhaps not, but it keeps server from calling data directly. It's its own distinct concept that has nothing to do with serving requests.

Reply:
The server package already calls `data` directly in two places: `Server.loadConfig`, and in `middleware.go`. This indirection hurts readability for no benefit.